Rakefile

```diff
@@ -78,7 +78,8 @@ namespace :test do
 
     # Increase the time limit from the default of 0.05 secs because coverage
     # may run slowly.
-    ENV["GCLOUD_TEST_COVERAGE_DEBUGGER_TIMEOUT"] ||= "0.3"
+    coverage_timeout = ENV["GCLOUD_TEST_COVERAGE_DEBUGGER_TIMEOUT"] || "0.3"
+    ENV["GCLOUD_TEST_DEBUGGER_TIMEOUT"] = coverage_timeout
```

Comment:
Are we going to set this somewhere for normal tests?

Reply:
yes, I've set this environment variable in the circleci settings.

Reply:
Okay.

@TheRoyalTnetennba can you make sure these get set in the kokoro configs?

```
GCLOUD_TEST_DEBUGGER_TIMEOUT=0.1
GCLOUD_TEST_COVERAGE_DEBUGGER_TIMEOUT=0.3
```